app/src/main/java/ch/epfl/sdp/appart/database/firestoreservicehelpers/FirestoreImageHelper.java

```diff
@@ -25,14 +28,51 @@ public FirestoreImageHelper() {
 
     @NotNull
     @NonNull
-    public CompletableFuture<Boolean> putImage(Uri uri, String name, String path) {
-        if (uri == null || name == null) {
+    public CompletableFuture<Boolean> putImage(Uri uri, String imagePathAndName) {
+        if (uri == null || imagePathAndName == null) {
             throw new IllegalArgumentException("parameters cannot be null");
         }
         CompletableFuture<Boolean> isFinishedFuture = new CompletableFuture<>();
-        StorageReference fileReference = storage.getReference(path).child(name);
+        StorageReference fileReference = storage.getReference().child(imagePathAndName);
```

Comment:
we can give the imagePathAndName as argument to `getReference` instead of using child

Reply:
Done!